gandalf.js

```diff
@@ -4,123 +4,128 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.